.gitignore

```diff
@@ -5,3 +5,5 @@
 /backend/target
 /frontend/node_modules
 */*.iml
+/backend/src/test/test.iml
+/backend/src/main/main.iml
```

Comment:
Diese .iml-Dateien sind ungewöhnlich. Habt ihr Module in diesen Ordnern angelegt?
Vermutlich könnt Ihr die imls löschen und hier auch die Zeilen in der gitignore weglassen.